features/volledigeNaam.feature

```diff
@@ -5,21 +5,28 @@ Functionaliteit: volledige naam
 Rule: de volledige naam wordt samengesteld door achter elkaar plaatsen van voornamen, voorvoegsels en geslachtsnaam
 
     Abstract Scenario: volledige naam van persoon
-        Gegeven de persoon met burgerservicenummer "{burgerservicenummer}" heeft de volgende gegevens:
-        | voornamen (02.10)                    | {voornamen}               |
-        | adellijke titel of predikaat (02.20) | {adellijkeTitelPredikaat} |
-        | voorvoegsel (02.30)                  | {voorvoegsel}             |
-        | geslachtsnaam (02.40)                | {geslachtsnaam}           |
-        Als persoon met burgerservicenummer "{burgerservicenummer}" wordt opgevraagd met fields=naam.volledigeNaam
-        Dan is in het antwoord naam.volledigeNaam gelijk aan "{volledigeNaam}"
+        Gegeven het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer   |
+        | <burgerservicenummer> |
+        En de persoon heeft de volgende naam gegevens
+        | voornamen (02.10) | adellijke titel of predikaat (02.20) | voorvoegsel (02.30) | geslachtsnaam (02.40) |
+        | <voornamen>       |                                      | <voorvoegsel>       | <geslachtsnaam>       |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                     |
+        | type                | ZoekMetBurgerservicenummer |
+        | burgerservicenummer | <burgerservicenummer>      |
+        | fields              | naam.volledigeNaam         |
+        Dan bevat de persoon met burgerservicenummer '<burgerservicenummer>' de volgende naam gegevens
+        | volledigeNaam   |
+        | <volledigeNaam> |
 
         Voorbeelden:
-        | burgerservicenummer | voornamen            | adellijkeTitelPredikaat | voorvoegsel | geslachtsnaam | volledigeNaam                               |
-        | 999995078           | Christina Maria      |                         |             | Maassen       | Christina Maria Maassen                     |
-        | 999994542           | Gerrit               |                         | den         | Braber        | Gerrit den Braber                           |
-        | 999994074           |                      |                         |             | Obbadah       | Obbadah                                     |
-
+        | burgerservicenummer | voornamen       | voorvoegsel | geslachtsnaam | volledigeNaam           |
+        | 999995078           | Christina Maria |             | Maassen       | Christina Maria Maassen |
+        | 999994542           | Gerrit          | den         | Braber        | Gerrit den Braber       |
+        | 999994074           |                 |             | Obbadah       | Obbadah                 |
 
+    @skip-verify
```

Comment:
deze skip je nu, waarschijnlijk omdat je testgevallen nu alleen beschrijven wat het zoeken (PersoonBeperkt) oplevert (zit ouders niet in). Kan je deze al omschrijven naar een vorm voor raadplegen op uuid?

Reply:
Ja, wil ik straks gaan implementeren